### ISSUE TITLE
Broken autofill: does not autofill / does not offer to autofill

### ISSUE THREAD

**Issue by Mugurell** [mozilla-mobile/fenix#28368]
Affects both address autofill and credit card autofill. Reproduces on current Playstore versions (108+).

### Steps to reproduce:
- Have at least one credit card added to "Settings" -> "Autofill" -> "Credit cards" and/or,
- Have at least one address added to "Settings" -> "Autofill" -> "Addresses",
- Access a website with forms for addresses and/or credit cards (eg: https://greenido.github.io/Product-Site-101/form-cc-example.html),
- Tap on a field to autocomplete such forms.

### Expected result:
- A new option to autocomplete the address/credit card appears in Fenix.
- Upon selecting an address/credit card the applicable fields are populated.

### Actual result:
- The option to autocomplete appears only once after install. Does not appear again even after an app restart.
- Upon selecting an address/credit card no applicable fields are populated.

### Additional info:
Issues seen on two devices
- OnePlus 7T Pro with Android 11
- Pixel 7 with Android 13
I expect the issue to affect all devices and current releases.

----- 

Showing a recording of the latest release - 108

https://user-images.githubusercontent.com/11428869/210341468-30a0458c-01c4-4140-badc-3568aa540c5a.mp4

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FNXV2-22216)

**Comment by Mugurell** [comment]
Seems like this is a know issue.
Closing as a duplicate of https://github.com/mozilla-mobile/fenix/issues/28173.
